certs/README.md

```diff
@@ -1,23 +1,23 @@
 # Certs Service
```

Comment:
Since certs service is going to be in Magistrala, we can change word SuperMQ to Magistrala in valid places

Reply:
We will probably keep certs as a part of IAM for SMQ.